### DISCUSSION TITLE
Pieces for Teams - shared snippet libraries, onboarding features, collaboration tools

### DISCUSSION THREAD

**Discussion by rosie-at-pieces** [pieces-app/support#59] [Ideas]
We've heard a lot of requests for this! Let us know what you think and any ideas you have.

**Comment by peaklabs-dev** [comment]
A team feature would be nice so we can collaborate with a team. This can be accomplished via workspaces-> Let us make shared workspaces where we can invite people into a workspace they only have access to set workspace they are invited too. For the shared workspace first the Wordpace feature https://github.com/pieces-app/support/discussions/13 has to be implemented and then the version history https://github.com/pieces-app/support/discussions/82 -> The version history tracks in detail what and who modified what in a snippet and also stores each version of the snippet
